controller/bin/daq_config.yaml

```diff
@@ -23,3 +23,5 @@ external_subnets:
   - subnet: 192.168.1.0/24
 
 site_path: /site
+
```

Comment:
mmmm... this change doesn't seem right. -- I was hoping to keep this file as somewhat of an example of how FOT should be configured, and this starts to break that.  Can this option be moved to a command-line flag for the specific test run where it's required?  Or, if specifically for this timeout test case, invert the "enabled" use below so you only specifically enable it for the DUT you want to timeout?